frontend/src/static/js/components/styles/PageSidebar.scss

```
@@ -240,9 +240,9 @@
 			width: 22px;
 			height: 22px;
 			border-radius: 50%;
-			background: #fff;
+			background: var(--bg-surface-raised);
 			// background: var(--sidebar-nav-border-color);
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)